paddle/fluid/operators/activation_op.cc

```diff
@@ -492,26 +492,6 @@ class SoftplusOpMaker : public framework::OpProtoAndCheckerMaker {
         "(bool, default false) Only used in cudnn kernel, need install cudnn.")
         .SetDefault(false)
         .AsExtra();
-    AddAttr<std::string>(
```

Comment:
Are we allowed to just delete deprecated parameters? I was thinking that due to backward compatibility they must stay as they were before

Reply:
Currently, there is a discussion and looking for a way to get rid of oneDNN-only attributes, which were used in fuse passes. Since we can get their values via `ExecutionContext`, we could slim base operators down https://github.com/PaddlePaddle/Paddle/issues/43232